core/conventional-commits/index.js

```diff
@@ -119,12 +119,31 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath }) {
         }
 
         log.verbose(type, "increment %s by %s", pkg.version, data.releaseType);
-        resolve(semver.inc(pkg.version, data.releaseType));
+        resolve({
+          currentVersion: pkg.version,
+          nextVersion: semver.inc(pkg.version, data.releaseType),
+        });
       });
     });
   });
 }
 
+function isBreakingChange(currentVersion, nextVersion) {
+  const releaseType = semver.diff(currentVersion, nextVersion);
+  switch (releaseType) {
+    case "major":
+      return true;
+    case "minor":
+      return semver.lt(currentVersion, "1.0.0");
```

Comment:
This will literally never work once a project has moved past v1.0.0. Same with the patch.

Reply:
Semver.diff(1.1.0,2.0.0) returns major. Why does it not work?

Reply:
But this is `semver.lt()`, how is that relevant?

Reply:
If current version is less than 1.0.0, then a minor bump should be treated the same as major one